roles/webserver/templates/unicorn.service.j2

```diff
@@ -8,6 +8,11 @@ Type=simple
 User=openfoodnetwork
 WorkingDirectory={{ current_path }}
 Environment=RAILS_ENV={{ rails_env }}
+
+{% if skylight_authentication is defined %}
+  Environment=SKYLIGHT_AUTHENTICATION={{ skylight_authentication }}
```

Comment:
We can check https://www.freedesktop.org/software/systemd/man/environment.d.html to set environment variables only in a one place

Reply:
Please open an issue to cover this later.